src/components/Layout/PageHeader.vue

```diff
@@ -60,7 +60,7 @@ export default {
         }),
 
         isLoggedIn() {
-            return this.user !== undefined;
+            return this.user;
```

Comment:
Why did you change current implementation?

Reply:
Because it blocks login/register page after logout.

Reply:
Ok, why?